airbyte-webapp/src/components/StreamTestingPanel/PageDisplay.module.scss

```diff
@@ -35,3 +35,16 @@
   max-height: 100%;
   overflow-y: auto;
 }
+
+// add a fade at the bottom of the tabPanel
+.tabPanel::after {
+  content: "";
+  position: absolute;
+  z-index: 1;
```

Comment:
We usually tend to maintain all z-indexes (since they are a global numbering system across all of the webapp) in the `_z-indices.scss` utility. Given that the purpose here is only to lift it slightly above the content, i.e. doing a "+1" I feel we can keep this in place here.